build.assets/build-package.sh

```diff
@@ -257,6 +259,7 @@ else
             # the GNUPG_DIR location here is assumed to contain a complete ~/.gnupg directory structure
             # with pubring.kbx and trustdb.gpg files, plus a private-keys-v1.d directory with signing keys
             # it needs to contain the "Gravitational, Inc" private key and signing key.
+            # we also use the rpm-sign/rpmmacros file instead which contains extra directives used for signing.
             EXTRA_DOCKER_OPTIONS="-v $(pwd)/rpm-sign/rpmmacros:/root/.rpmmacros -v $(pwd)/rpm-sign/popt-override:/etc/popt.d/rpmsign-override -v ${GNUPG_DIR}:/root/.gnupg"
```

Comment:
Potentially append to `EXTRA_DOCKER_OPTIONS` instead of overriding the previous value -- while not a big difference it means the signed and unsigned behavior is more likely to remain consistent when a future maintainer has to muck with it.

Reply:
I did try this but you can only use one `~/.rpmmacros` file, so you have to combine the two into one and then you end up in a weird situation with `cat` not adding line breaks etc. I figured it was easier overall to just have two separate files, one for unsigned (which is only ever used for local testing) and one for signed.

Reply:
Ah yeah, I missed that `rpm-sign/rpmmacros` and `rpm/rpmmacros` were different paths.